package.json

```diff
@@ -9,7 +9,8 @@
     "web": "expo start --web",
     "test": "jest",
     "build": "eas build -p android",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "start:wsl": "REACT_NATIVE_PACKAGER_HOSTNAME=$(netsh.exe interface ip show address 'Ethernet' | grep 'IP Address' | sed -r 's/^.*IP Address:\\W*//') expo start"
```

Comment:
Organize the scripts logically. For example all different kinds of start scripts should be together.